deploy/values.yaml

```diff
@@ -38,32 +38,32 @@ application:
       tmp:
         mountPath: /tmp
 
-    resources: 
+    resources:
       limits:
-        memory: 1Gi 
+        memory: 1Gi
         cpu: 0.6
       requests:
         memory: 128Mi
         cpu: 0.1
 
     readinessProbe:
       enabled: true
-      failureThreshold: 5
-      periodSeconds: 5
+      failureThreshold: 50
```

Comment:
we can update probes for values-local and not here since probes are an issue on snos only.

Reply:
for review, probes were already there, I just changed the values.